client/kubernetes/src/test/java/org/jolokia/kubernetes/client/ManualConnectionTest.java

```diff
@@ -0,0 +1,33 @@
+package org.jolokia.kubernetes.client;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+/**
+ * This is a utility to attempt to connect to manually test connection to a JVM running in kubernetes
+ *  1. Ensure that your current kubernetes context is set to the context you attempt to use
+ *     (e.g. kubectl config use-context my-context)
+ *  2. Ensure that your kubernetes config contains valid credentials for the k8s cluster of that context
+ */
+public class ManualConnectionTest {
```

Comment:
Renamed file to be able to remove the suppresswarnings. It is debatable how useful this manual test is. It might have been a main method as well. However running as a test from the IDE with the ability to use assertions etc. can be nice.

Reply:
+1 for keeping it as test instead if `main()`. If the warnings are gone, I'm fine ;)